_components/footer/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/footer/accessibility-tests/
+layout: accessibility-test
+component:
+  name: footer
+title: Footer accessibility tests
+category: Components
+lead: Any non-interactive USWDS footer component should pass these manual accessibility tests.
```

Comment:
@mahoneycm 
**issue:** Can we remove "non-interactive" here? Wondering if it might have been copied over from the [tag page](https://designsystem.digital.gov/components/tag/accessibility-tests/).
```suggestion
lead: Any USWDS footer component should pass these manual accessibility tests.
```

Reply:
Yep sure was! Great catch here